resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/LoggingPodStatusWatcher.scala

```diff
@@ -177,4 +190,34 @@ private[k8s] class LoggingPodStatusWatcherImpl(
   private def formatTime(time: String): String = {
     if (time != null ||  time != "") time else "N/A"
   }
+
+  override def watchOrStop(sId: String): Boolean = if (waitForCompletion) {
+    logInfo(s"Waiting for application ${conf.appName} with submission ID $sId to finish...")
```

Comment:
Oh, it's a compilation error.
```scala
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/LoggingPodStatusWatcher.scala:195: object appName is not a member of package conf
[error]     logInfo(s"Waiting for application ${conf.appName} with submission ID $sId to finish...")
[error]                                              ^
[error] one error found
```

Reply:
Can you confirm that this is the way you want it?